storage/rocksdb/ha_rocksdb.cc

```diff
@@ -6034,6 +6036,45 @@ static bool rocksdb_user_table_blocked(legacy_db_type db_type) {
   return db_type == DB_TYPE_INNODB;
 }
 
+static bool rocksdb_is_supported_system_table(const char *db_name,
```

Comment:
how about these "missing" system table(such as mysql.temp_user), still use innodb instead?

Reply:
Any table not listed here will use InnoDB, but, unless something fell through the cracks, that should be only the Enterprise Edition tables (MEB, tp plugin), which should not apply. What is the `mysql.temp_user` table?

Reply:
> What is the mysql.temp_user table?

https://github.com/facebook/mysql-5.6/blob/fb-mysql-8.0.32/sql/dd/impl/system_registry.cc#L254

The reason I had asked this is because the list is different from https://github.com/facebook/mysql-5.6/pull/1390.. I am changing my PR to also use exactly same list..

Reply:
just rebased my PR based on this PR(will rebase again after this PR is checked in)

https://github.com/facebook/mysql-5.6/pull/1390/commits/7db177936be7af489b530c758f9ae6650cb3376a#diff-773930212c131b69bdf9e5944d12142364419730bb32c19a1266dedd77aa72f8

Reply:
> > What is the mysql.temp_user table?
> 
> https://github.com/facebook/mysql-5.6/blob/fb-mysql-8.0.32/sql/dd/impl/system_registry.cc#L254

Oh, I see. As far I can tell these are temporary test tables created by MTR tests in mysql schema. I think it's fine to convert them once i.e. InnoDB is disabled, because at that point we'll be able to confirm that there are indeed no other unconverted system tables left, and the conversion of these should be trivial. If I'm right that they are temporary then there's no need to handle their upgrade neither.

Reply:
yep. sounds good. we can convert them during disable innodb time